system-variables.md

```diff
@@ -791,7 +792,52 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
         ERROR 1062 : Duplicate entry '1' for key 'PRIMARY'
         ```
 
-Constraint checking is always performed in place for pessimistic transactions (default).
+### `tidb_constraint_check_in_place_pessimistic` <span class="version-mark">New in v6.3.0</span>
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Default value: `ON`
```

Comment:
```suggestion
- Default value: `1`
```
To be consistent with the Chinese version. I'm tired of changing the boolean literals of system variables though 😢 .